### ISSUE TITLE
Start minecraft server jar with same java version as starter jar

### ISSUE THREAD

**Issue by sammyt291** [BloodyMods/ServerStarter#38]
Start minecraft server jar with same java version as starter jar

An option to do the following: (Or preferably by default)
If serverstarter is ran with java8, it should then run the server with the same java8, not default to the newest.
this would help with compatability with certain panels such as WiSP which autodetect the java version and supply all 3 main java versions to the container, 8, 11 and 16.

Ive had some users using serverstarter having issues, as WiSP detects java8 correctly, and runs serverstarter with 8.
but serverstarter finds 16 is available and uses that anyway, which the pack its running doesnt support.

would be great to have it use the java it has itself been ran with, or at the very least; an option to do so.

All the best,
Sam.

**Comment by 0xpr03** [comment]
Same problem here, started with java 8 it then proceeds with the server to java 17..

**Comment by BloodWorkXGaming** [comment]
There is an Option to force the path of java to use in the config `forcedJavaPath`, however it isn't as convenient as reusing the same jvm of course.

**Comment by 0xpr03** [comment]
Yeah I sadly had to discover that the modpack author didn't even update to config v2 where forcedJavaPath is supported. So I went ahead and started it manually with all the JVM args, like I always did ;)

**Comment by BloodWorkXGaming** [comment]
2.3.0 allows supported java version to be set in the config now, which will be found on the $PATH
